prody/proteins/ciffile.py

```diff
@@ -226,7 +226,7 @@ def _parseMMCIFLines(atomgroup, lines, model, chain, subset,
             fieldCounter += 1
             fields[line.split('.')[1].strip()] = fieldCounter
 
-        if line.startswith('ATOM') or line.startswith('HETATM'):
+        if line.startswith('ATOM  ') or line.startswith('HETATM'):
```

Comment:
Should it be ATOMS or ATOM here? just wondering

Reply:
Should be ATOM with two spaces or HETATM. These are the two options.